src/typeDefs/inputs.ts

```diff
@@ -138,7 +138,7 @@ export const inputs = gql`
   input OrganizationInput {
     name: String!
     description: String!
-    location: String
+    address: AddressInput
```

Comment:
@Anubhav-2003 make it non nullable in the input

Reply:
None to make it easier for the admin, but `Country Code` should come from a list of enums when updated

Reply:
@xoldyckk Sir, I have made it non-nullable in my last commit.

Reply:
@palisadoes  @xoldyckk Sir, I think in the front-end we should use an API to get the country code, and then use enums to verify it in the backend. There has to be major changes in the Talawa Admin, from updating queries, mutations, to using API's to send the data to the backend before this PR can be merged. Otherwise the Talawa-Admin portal would break.